libcheckout

```diff
@@ -49,6 +48,7 @@ function checkout::use_cache() {
     checkout::clone
   fi
 
+  # shellcheck disable=SC2181
```

Comment:
Can you put an explanation why we are turning off checks?

Reply:
Done.